client/src/redux/donation-saga.js

```diff
@@ -68,21 +68,21 @@ function* postChargeStripeSaga({ payload }) {
     const err =
       error.response && error.response.data
         ? error.response.data.error
-        : defaultDonationError;
+        : defaultDonationErrorMessage;
     yield put(postChargeStripeError(err));
   }
 }
 
 function* postChargeStripeCardSaga({ payload }) {
   try {
-    yield call(postChargeStripeCard, payload);
+    const { error } = yield call(postChargeStripeCard, payload);
+    if (error) throw error;
     yield put(postChargeStripeCardComplete());
   } catch (error) {
-    const err =
-      error.response && error.response.data
-        ? error.response.data.error
-        : defaultDonationError;
-    yield put(postChargeStripeCardError(err));
+    const errorMessage = error.message
+      ? error.message
+      : defaultDonationErrorMessage;
```

Comment:
Just felt like making this more succinct 🤷‍♂️ 
```suggestion
    const errorMessage = error.message || defaultDonationErrorMessage;
```